course/src/isometry.cc

```diff
@@ -0,0 +1,248 @@
+#include <isometry.h>
+
+namespace ekumen {
+  namespace math {
+
+  //Vector3
+
+  const Vector3 Vector3::kUnitX(1., 0., 0.);
+  const Vector3 Vector3::kUnitY(0., 1., 0.);
+  const Vector3 Vector3::kUnitZ(0., 0., 1.);
+  const Vector3 Vector3::kZero(0., 0., 0.);
+
+  Vector3 Vector3::operator += (const Vector3 & obj) const
+  {
+    return Vector3( x_ + obj.x(), y_ + obj.y(),z_ +  obj.z());
```

Comment:
This does not do the assignment part of the operation, since it does not change the current object.